src/utils/constants.ts

```diff
@@ -5,6 +5,7 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 export const MINIMUM_REQUIRED_VERSION_CORE_EXTENSION = '58.4.1';
-export { CORE_EXTENSION_ID, SFDX_PROJECT_FILE } from 'mobile-lsp-server';
+export const CORE_EXTENSION_ID = 'salesforce.salesforcedx-vscode-core';
+export const SFDX_PROJECT_FILE = 'sfdx-project.json';
```

Comment:
These constants, too, were originally defined in root, which moved to LSP server and then reverse exported back to the root project. Since we're consolidating they're back to where they were.